docs/api/commands/request.mdx

```diff
@@ -364,6 +364,13 @@ functions as if it came from the browser.
 The intention of `cy.request()` is to be used for checking endpoints on an
 actual, running server without having to start the front end application.
 
+### User agent
+
+Trying to change the `User-Agent`?
+
+See
+[how to override the default user agent the browser uses](../../faq/questions/using-cypress-faq#Can-I-override-or-change-the-default-user-agent-the-browser-uses).
```

Comment:
the "../.." step to create the relative path isn't necessary in the Docs site. You can use: 

```suggestion
[how to override the default user agent the browser uses](/faq/questions/using-cypress-faq#Can-I-override-or-change-the-default-user-agent-the-browser-uses).
```